src/RA_Integration.vcxproj

```diff
@@ -179,6 +191,7 @@
     <ProjectName>RA_Integration</ProjectName>
     <ProjectGuid>{825CB292-F069-4B27-82CF-3417746ACDF3}</ProjectGuid>
     <RootNamespace>RA_Integration</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```

Comment:
This is the lowest version of the Windows SDK I have in `Windows Kits`. Someone else might need a different version. Recommend using 10.0.15063+. It can possibly work with 8.1 or 7.1 but haven't tested it.